<launch>
   <include file="$(find rcvm_aqua)/launch/rcvm.launch"/>
	<node pkg="rcvm_aqua" type="indicate_movement_left.sh" name="indicate_movement"/>
</launch>

